lib/pure/os.nim

```diff
@@ -2405,10 +2405,9 @@ proc moveDir*(source, dest: string) {.tags: [ReadIOEffect, WriteIOEffect], noWei
   ## * `existsOrCreateDir proc <#existsOrCreateDir,string>`_
   ## * `createDir proc <#createDir,string>`_
   if not tryMoveFSObject(source, dest):
-    when not defined(windows):
-      # Fallback to copy & del
-      copyDir(source, dest)
-      removeDir(source)
+    # Fallback to copy & del
+    copyDir(source, dest)
```

Comment:
this fix doesn't seem like it'll work, because `tryMoveFSObject` raises on windows instead of returning false; for proper fix, see what i suggested in https://github.com/nim-lang/Nim/issues/18216#issuecomment-857465533 (it requires also changing `tryMoveFSObject`)